### PR TITLE
New article: Add domain in Office 365

### PR DESCRIPTION
**Pull request by DonNateR**
Part of PR #2139

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-de5a1e5ecd/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3c7056e8ff/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-8d75642cb0/how-to/). :bow:

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Edits attached. 

[Add a domain in Office 365_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1132266/Add.a.domain.in.Office.365_editsKH.docx)
[Add a domain in Office 365_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1132267/Add.a.domain.in.Office.365_editsKH.pdf)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-428de2c59b/how-to/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-de90184f7d/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d0c84a191d/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-31eeac0113/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5e1882b5f5/how-to/).

File: content/office-365/add-a-domain-in-office-365.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+4. In the **Domains** section, click **Add Domain**.
+5. On the next page enter the domain name that you want to add.
+
+   <img src="{% asset_path office-365/add-a-domain-in-office-365/add-domain-in-o365-sc2.png %}" />
+
+6. Verify that you own the domain.
+
+   This step requires you to log in to your domain's DNS host portal. If you do not know who your DNS host is, see [Find your DNS host](/how-to/find-your-dns-host/).
+
+    1. Copy the TXT record information provided in the Office 365 portal
```
Comment (stephamon) [inline content/office-365/add-a-domain-in-office-365.md]: Steps 1, 2, and 3 need periods at the end of the sentence.

File: content/office-365/add-a-domain-in-office-365.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    4. Save this change at your DNS host.
+
+    **Note:** The TXT record could take 24-48 hours to be verified by Office 365.
+
+    After your domain has been successfully verified by Office 365, Office 365 prompts you to set up your online services.
+
+7. To set up your online services, select one of the following options:
+
+    - **Set up my online services for me**: Inputs all DNS records from your DNS host automatically. All mail associated with your DNS host is immediately reroute to Office 365.
```
Comment (stephamon) [inline content/office-365/add-a-domain-in-office-365.md]: "reroute" should be "rerouted"

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-da8701e31b/how-to/).

**Comment by stephamon** [comment]
👍

**Pull request merged**